.circleci/ci/src/jobs/job-release-commit-and-prepare-next-version.ts

```diff
@@ -82,6 +79,10 @@ sed -i 's#version: ".*"#version: "${nextVersion}${nextQualifier}-SNAPSHOT"#' gra
 sed -i 's#"version": ".*"#"version": "${nextVersion}${nextQualifier}-SNAPSHOT"#' gravitee-apim-console-webui/build.json
 sed -i 's#"version": ".*"#"version": "${nextVersion}${nextQualifier}-SNAPSHOT"#' gravitee-apim-portal-webui/build.json
 
+# Helm chart increase version and clean the artifacthub.io/changes annotation
+sed "0,/version.*/s/version.*/version: ${nextVersion}${nextQualifier}/" -i helm/Chart.yaml
+sed -e '/artifacthub.io\\/changes/q' Chart.yaml > Chart2.yaml && mv Chart2.yaml Chart.yaml
```

Comment:
Why not editing the file directly with `-i` ?
And what is this line for ?

Reply:
This is for the point 2: clean all after artifacthub.io changes. I'll replace by -i